### PR TITLE
lkft: Linux updates for 4.18

### PR DESCRIPTION
**Pull request by mrchapp**
LKFT Linux changes due to 4.18 being released.

### CODE REVIEW

File: recipes-overlayed/kselftests/kselftests-mainline_4.18.bb [hunk 1]
```diff
@@ -9,13 +9,13 @@ SRC_URI += "\
     file://0001-selftests-lib-allow-to-override-CC-in-the-top-level-Makefile.patch \
     file://0001-selftests-gpio-fix-build-error.patch \
     file://0001-selftests-gpio-use-pkg-config-to-determine-libmount-.patch \
-    file://0001-selftests-net-use-LDLIBS-instead-of-LDFLAGS.patch \
+    file://0001-selftests-net-use-LDLIBS-instead-of-LDFLAGS-next-20180416.patch \
     file://0002-selftests-seccomp-use-LDLIBS-instead-of-LDFLAGS.patch \
```
Comment (danrue) [inline recipes-overlayed/kselftests/kselftests-mainline_4.18.bb]: Why this change?

Comment (mrchapp) [reply recipes-overlayed/kselftests/kselftests-mainline_4.18.bb]: Old version of the patch doesn't apply cleanly. Around 4/16 the selftest/net/Makefile changed, so the patch had to be updated for linux-next. Now it's landed in mainline.
